gradlew

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 
 #
-# Copyright � 2015-2021 the original authors.
+# Copyright © 2015-2021 the original authors.
```

Comment:
The changes in `gradlew` look fine. I'm surprised the file wasn't already UTF-8.

Reply:
Since this is a 3rd-party file, it's probably best to fix it separately the next time we update it, though.

Reply:
good point.

Reply:
So given that this is unrelated, please revert this file.

Reply:
reverted.  we do want to fix it, right?

Reply:
Yes, we can do it the next time we update gradle (which will be in the JavaFX 20 time frame).